src/script/ceph-release-notes

```diff
@@ -110,6 +110,30 @@ def split_component(title, gh, number):
         else:
             return 'UNKNOWN: ' + title
 
+def _title_message(commit, pr, strict):
+    title = pr['title']
+    message_lines = commit.message.split('\n')
+    if strict or len(message_lines) < 1:
+        return (title, None)
+    lines = []
+    for line in message_lines[1:]:
+        if 'Reviewed-by' in line:
```

Comment:
while you're at it, can you add this line too:
```py
reviewed=re.compile(r"Rev(.*)By", re.IGNORECASE)
if reviewed.match(line):
```

otherwise we'll see cases where reviewed-by has some typos etc. and the script will not ignore the line